opsdroid/database/mongo/__init__.py

```diff
@@ -77,9 +82,14 @@ async def get(self, key):
 
         """
         _LOGGER.debug("Getting %s from MongoDB.", key)
-        return await self.database[key].find_one(
-            {"$query": {}, "$orderby": {"$natural": -1}}
-        )
+        if self.query_data:
+            return await self.database[key].find_one(
+                {"$query": self.query_data, "$orderby": {"$natural": -1}}
+            )
+        else:
+            return await self.database[key].find_one(
+                {"$query": {}, "$orderby": {"$natural": -1}}
+            )
```

Comment:
If the `query_data` was set, then use that for querying, otherwise just default to the previous way.